src/app/interfaces/strongRoute.ts

```diff
@@ -66,9 +76,10 @@ export class StrongRoute {
    * Add a child route
    * @param name Route name
    * @param config Additional router configurations
+   * @param isRoot Is this a root category
    */
-  add(name: string, config: Partial<Route> = {}) {
-    return new StrongRoute(this, name, config);
+  add(name: string, config: Partial<Route> = {}, isRoot?: boolean) {
```

Comment:
why are we exposing `isRoot` publically?

Reply:
Maybe the variable name is a little misleading, it is designed for child categories such as the Sites Category which allows it to set itself as the root, even though the Projects Category is the actual root. Basically it lets me stop the child category redefining the parent category routes and vice versa. This is part of the solution for #159

Reply:
Responded to #159